enhancements/windows-containers/health-management.md

```diff
@@ -168,6 +181,10 @@ When making use of a PowerShell script, retries should be present within the scr
 is waiting for the HNS network to be created in order to create an HNS endpoint. In that specific case, the PowerShell
 script which creates the endpoint should include a retry period which waits for the needed Network to exist.
 
+Services marked with `bootstrap: true` will start before any others. Bootstrap services cannot depend on a non-bootstrap
+service; WICD will throw an error if this is detected. Similarly, each service that has the bootstrap flag set as true
+must have a higher priority than all non-bootstrap services. WICD will throw an error if this is not the case.
+
```

Comment:
I would also add that there should be no overlap in the priorities of bootstrap and controller services.

Reply:
I think that's implied already, but sure I can state it explicitly.